spectator-ext-placeholders/build.gradle

```diff
@@ -1,3 +1,8 @@
 dependencies {
   compileApi project(':spectator-api')
+
+  // The MdcTagFactoryTest requires a logging implementation that supports the MDC
+  // functionality, which the slf4j-nop (default) and slf4j-simple implementations do not.
```

Comment:
You get no tags from the MdcTagFactory as the underlying `MDC.get()` call always returns null.